### ISSUE TITLE
Media not showing up on IA after upload

### ISSUE THREAD

**Issue by foundscapes** [OpenArchive/OpenArchive#14]
![screenshot_2015-04-10-11-32-36](https://cloud.githubusercontent.com/assets/2231711/7095686/3591cdce-df81-11e4-80d2-d2a82f0cc078.png)

**Comment by vitriolix** [comment]
the file will showup eventually.  Unfortunately IA takes some time to process and their page doesn't really make that clear.  We should probably popup a message once your upload complete to set user expectations.

**Comment by foundscapes** [comment]
Yeah, something like that would be better than "view media online", thanks
